kubeflow/common/asm/Makefile

```diff
@@ -3,16 +3,22 @@
 # For example: 1.9.2-asm.1+config4:install_asm_1.9.2-asm.1-config4
 # The part before colon symbol should be used for ASM_PACKAGE_VERSION.
 # The part after colon symbol should be used for INSTALL_ASM_SCRIPT_VERSION.
-# IMPORTANT: You must change corresponding kpt value in kubeflow/env.sh if you upgrade 
-# For example: export ASM_LABEL=asm-193-2
-# Then go back to kubeflow/ directory, run the following commands:
-# 1. source ./env.sh
-# 2. bash ./kpt-set.sh
-# 3. make apply
-# The above commands allow you to set new namespace labels and re-apply your resources.
 ASM_PACKAGE_VERSION=1.9.2-asm.1+config4
 INSTALL_ASM_SCRIPT_VERSION=install_asm_1.9.2-asm.1-config4
-
+# ASM Upgrade Guide
+# 1: Determine the ASM version you want to upgrade, follow the instruction above
+#     to update ASM_PACKAGE_VERSION and INSTALL_ASM_SCRIPT_VERSION. Then run
+#     `make install-asm` to install new ASM version.
+# 2: Update kubeflow/common/asm/env.sh for the ASM version, this looks like 
+#     asm-192-1, and you can find the exact value in istiod pod's label as well.
+#     For example: You can find label `istio.io/rev: asm-192-1` in istiod-asm-192-1
+#     service. Then run `source env.sh` under `common/asm/` folder.
+# 3: Run `bash kpt-set.sh` under `common/asm` folder. It will configure kpt setter
+#     value for all components which requires to set ASM namespace label.
```

Comment:
I agree that it can be simplified to one line of shell call. The reason I write this in a shell script is that:

1. The root env.sh and kpt-set.sh will need them for full fledged deployment
2. Keeping track of kpt setter and environment variables in version control style (git). 

I think education of `kpt` will need its own section in Kubeflow Google Cloud instruction. The full fledged Kubeflow deployment guide serves a single purpose: Allows user to deploy a working solution of Kubeflow. Once they have a cluster ready, it will make their `kpt` learning much easier since they can compare the effect of their kpt commands based on a working deployment. Does it make sense?